processing/src/main/java/io/druid/query/groupby/epinephelinae/GroupByQueryEngineV2.java

```diff
@@ -402,6 +406,29 @@ public void close()
     }
   }
 
+  private static void convertRowTypesToOutputTypes(List<DimensionSpec> dimensionSpecs, Map<String, Object> rowMap)
+  {
+    for (DimensionSpec dimSpec : dimensionSpecs) {
+      Object baseVal = rowMap.get(dimSpec.getOutputName());
```

Comment:
I decided to keep it as-is, the function for compute(...) would need to lookup the outputType associated with the dimension name key. 

That info is only kept in the DimensionSpecs list from the query in that context, so it would have to scan the list for every key or keep some other mapping structure of dimName->outputType around